### PR TITLE
implement product details in payment apis

### PR DESCRIPTION
**Pull request by robrap**
ARCH-856

### CODE REVIEW

File: ecommerce/extensions/basket/views.py [hunk 1]
```diff
@@ -505,6 +498,14 @@ def get_serialized_basket(self, context):
             ],
         }
 
+        for line_data in context['lines_data']:
+            serialized_basket['products'].append({
+                'sku': line_data['sku'],
+                'title': line_data['product_title'],
```
Comment (robrap) [inline ecommerce/extensions/basket/views.py:504]: Reviewers: I renamed `name` to `title` to better match the original names, but this is breaking the frontend.  Can someone please take this over and make that frontend PR as well?  Thank you!

**Review by albemarle — APPROVED** [review]

File: ecommerce/extensions/basket/views.py (hunk 1 above)
Comment (robrap) [reply ecommerce/extensions/basket/views.py:504]: FYI: @julianajlk will be taking care of the frontend change.

**Comment by robrap** [comment]
@edx/ecommerce: FYI: This is also basically just messing with the new api endpoints, and should be safe to merge/deploy.  I'm going OOO for a couple of days, so someone else on my team may take care of it, but they may not push it to Prod.

**Comment by albemarle** [comment]
The front-end PR is https://github.com/edx/frontend-app-payment/pull/40 .

**Pull request merged**